src/test/java/de/bonndan/nivio/input/KubernetesTest.java

```diff
@@ -52,20 +53,26 @@ void setup() {
     }
 
     @Test
-    public void testRead() throws IOException {
+    public void testRead() {
```

Comment:
Just added a comment here, for confirmation .you have removed IOException here. are you sure?